src/panels/lovelace/common/compute-unused-entities.ts

```diff
@@ -72,11 +72,24 @@ export const computeUnusedEntities = (
   config: LovelaceConfig
 ): string[] => {
   const usedEntities = computeUsedEntities(config);
-  return Object.keys(hass.states)
-    .filter(
-      (entity) =>
-        !usedEntities.has(entity) &&
-        !EXCLUDED_DOMAINS.includes(entity.split(".", 1)[0])
-    )
-    .sort();
+  const allEntites = new Set(Object.keys(hass.states));
```

Comment:
Let's do this calculation inside `calcUnusedEntities` and so just pass `hass`.

Reply:
Gotcha. Didn't think it was necessary to send the whole hass object but I don't think there is any drawback